src/App.vue

```diff
@@ -77,6 +77,15 @@ export default {
     }
   },
   computed: {
+    remainCount () {
+      let leftCount = 0;
```

Comment:
한줄로
```suggestion
return this.todoItems.filter(_ => !_.completed).length
```
와 같이 할 수 있지 않을까요?